.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules
 ncp-debug.log
 npm-debug.log
 app/utils/config.js
+.idea
```

Comment:
Ah, this doesn't really belong here, but in your _global_ gitignore file. Project .gitignore files should refer to project-specific files (e.g. output files that we'll never want to check in).